<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="elevatorControlSystemContainer">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Elevator Control System Control</string>
    </property>
    <widget class="QGroupBox" name="ecsTriggers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>191</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Triggers</string>
     </property>
     <widget class="QPushButton" name="ecsFire">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Fire</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ecsPowerOutage">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Power Outage</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="floorPassengerContainer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Passenger Control</string>
    </property>
    <widget class="QComboBox" name="passengerNumberSelector">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="passengerNumberTag">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Passenger Number</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="floorContainer">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>311</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Floor Control</string>
     </property>
     <widget class="QGroupBox" name="floorPanelContainer">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>120</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>floorPanel</string>
      </property>
      <widget class="QPushButton" name="floorPanelUpButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>83</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
      <widget class="QPushButton" name="floorPanelDown">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>83</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="floorNumberSelector">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>86</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="floorNumberTag">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Floor Number</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="elevatorContainer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>331</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Elevator Control</string>
    </property>
    <widget class="QPushButton" name="elevatorDest5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elevatorDest3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elevatorDest6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elevatorDest4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elevatorDest1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elevatorDest2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elevatorOpen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elevatorClose">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elevatorHelp">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>224</red>
           <green>27</green>
           <blue>36</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>224</red>
           <green>27</green>
           <blue>36</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>224</red>
           <green>27</green>
           <blue>36</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QComboBox" name="elevatorSelector">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevator Number</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elevatorDest7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="scenarioContainer">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Scenarios</string>
    </property>
    <widget class="QPushButton" name="scenarioSucess">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Success 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scenarioSuccess2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Success 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scenarioOverload">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Overload</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scenarioObstruction">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Obstruction</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scenarioDefault">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
